components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -8,32 +8,41 @@ import { Notification, Type } from 'app/entities/notifications/notification.mode
 
 import * as fromLayout from './layout.reducer';
 import { MenuItemGroup } from './layout.model';
-import { sidebarMenuGroups, showPageLoading } from './layout.selectors';
-import { ShowPageLoading, UpdateSidebarMenuGroups } from './layout.actions';
+import { sidebar, showPageLoading } from './layout.selectors';
+import { ShowPageLoading } from './layout.actions';
 
 // Important! These must match components/automate-ui/src/styles/_variables.scss
 enum Height {
   Navigation = '70px',
   Banner = '110px'
 }
 
+export enum Sidebar {
+  Dashboards = 'dashboards',
+  Applications = 'applications',
+  Infrastructure = 'infrastructure',
+  Compliance = 'compliance',
+  Settings = 'settings',
+  Profile = 'profile'
+}
+
 @Injectable({
   providedIn: 'root'
 })
 export class LayoutFacadeService {
   headerHeight = '70px';
```

Comment:
Is this default headerHeight because by default we are not dealing with any notifications?  If that is the case, my thought is to use `Height.Navigation` so that there's only one location to modify.

Reply:
@SEAjamieD Thanks for catching that! Updated👍🏻